index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main id="main">
+     <h1 id="title">MUHAMMED ALI</h1> 
+      <p>The American professional boxer</p>
+      <section id="tribute-info">
+        <h3 id="headline">Here's a time line of Muhammed life:</h3>
+        <ul>
+          <li><strong>1914</strong> - he was known for his social message of black pride and black resistance to white domination and for refusing induction into the U.S. Army during the Vietnam War.24 Aug 2022</li>
+          <li>
+            <strong>1933</strong> - Leaves his family's farm to attend the
+            University of Minnesota, thanks to a Depression era program known as
+            the "National Youth Administration"
+
+          <li>
+            <strong>1935</strong> - Has to stop school and save up more money.
+            Works in the Civilian Conservation Corps, helping starving
+            Americans. "I saw how food changed them", he said. "All of this left
+            scars on me."
+          </li>
+          <li>
+            <strong>1937</strong> - Finishes university and takes a job in the
+            US Forestry Service
+          </li>
+          <li>
+            <strong>1938</strong> - Marries wife of 69 years Margret Gibson.
+            Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman,
+            he returns to school study under Stakman, who teaches him about
+            breeding pest-resistent plants.
+          </li>
+       
+            <strong>1941</strong> - Tries to enroll in the military after the
+            Pearl Harbor attack, but is rejected. Instead, the military asked
+            his lab to work on waterproof glue, DDT to control malaria,
+            disinfectants, and other applied science.
+    
```

Comment:
- [x] Kindly add the missing li opening and closing tags in this block.

Reply:
@Ofarouq310, You have used a very polite word to request changes 💯  Kindly state why it is necessary to have a closing tag for all your HTML elements ✌️